### ISSUE TITLE
flowcharts and templates 4 research

### ISSUE THREAD

**Issue by LeeRuns** [CHC-CSU/Team-CHC-CSU#27]
[article review template.docx](https://github.com/CHC-CSU/Team-CHC-CSU/files/125526/article.review.template.docx)

**Comment by LeeRuns** [comment]
![image.png](https://overvio-usercontent.s3.eu-central-1.amazonaws.com/image-035d2a7d7e9b99c37a6ee5b9cfbde676e918be0fa61def24676dd19c3d4fb78c.png)

**Comment by LeeRuns** [comment]
i'd look at the wikipedia page for non-newtonian fluids they divide up types pretty well

**Comment by LeeRuns** [comment]
oooo found a [list of all of the solvers in open foam](http://www.openfoam.org/features/standard-solvers.php)

**Comment by ncommarato** [comment]
![fluid_tly](https://cloud.githubusercontent.com/assets/16928098/13019000/ff836278-d189-11e5-9c20-51916401a7ea.png)

**Comment by ncommarato** [comment]
@LeeRuns are we looking into building a flow chart like this? Because I can work on something like this that might be help for our presentation. We could individually talk about the solvers used for each and we are focus is.

**Comment by LeeRuns** [comment]
yes the flowchart and expanding the table shown in my comments above.  nice find

**Comment by LGintner** [comment]
@LeeRuns found this article on openfoam modeling of viscoelastic fluids in 3D using a few different solvers.

[Numerical simulation of three-dimensional viscoelastic planar contraction flow using the software OpenFOAM.docx](https://github.com/CHC-CSU/Team-CHC-CSU/files/131453/Numerical.simulation.of.three-dimensional.viscoelastic.planar.contraction.flow.using.the.software.OpenFOAM.docx)

**Comment by LeeRuns** [comment]
@LGintner  i have a meeting with jeff later today, do you have a draft of the table of different fluids and solvers

**Comment by LGintner** [comment]
@LeeRuns I am working in it tonight. did that paper that i found help you? It used various solvers for openfoam specifically.

**Comment by LGintner** [comment]
@LeeRuns I am working in it tonight. did that paper that i found help you? It used various solvers for openfoam specifically.

**Comment by LeeRuns** [comment]
yes, it did.   
also the summary made it really easy to understand what exactly the paper covers.

good to see that they were able to test the model and that it was successful

@ncommarato @LGintner @zpeek @mlreid358 
I talked to Jeff today.  He said we should try to just focus on information related to shear thinning fluids that are time independed. (psudoplastic).Lets try and prioritize those 1st.  Hope this is helpful at trimming down the scope

**Comment by LeeRuns** [comment]
found this explaination of different solvers u can use for non-newtonian fluids.  I see some methods that are in openfoam under the rheology sets.  i don't actually get how that part goes into the cfd.  
[paper on non-newtonian mathmatical models](http://www1.karlin.mff.cuni.cz/~malek/new/images/Lecture4.pdf)
